rules/integrations/o365/impact_microsoft_365_mass_download_by_a_single_user.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2021/07/15"
+maturity = "development"
+updated_date = "2021/07/15"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when Microsoft Cloud App Security reported when a single user performs more than 50 downloads within 1 minute.
+"""
+false_positives = ["Unknown"]
+from = "now-30m"
+index = ["filebeat-*", "logs-o365*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Microsoft 365 - Mass download by a single user"
+note = """## Config
+
+The Microsoft 365 Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule.
+"""
+references = [
+    "https://docs.microsoft.com/en-us/cloud-app-security/anomaly-detection-policy",
+    "https://docs.microsoft.com/en-us/cloud-app-security/policy-template-reference",
+]
+risk_score = 47
+rule_id = "571ff456-aa7f-4e48-8a88-39698bb5418f"
+severity = "medium"
+tags = ["Elastic", "Cloud", "Microsoft 365", "Continuous Monitoring", "SecOps", "Configuration Audit"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.dataset:o365.audit and event.provider:SecurityComplianceCenter and event.category:web and event.action:"Mass download by a single user" and event.outcome:success
+'''
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+name = "Data Destruction"
+reference = "https://attack.mitre.org/techniques/T1485/"
+id = "T1485"
+
+[rule.threat.tactic]
+name = "Impact"
+reference = "https://attack.mitre.org/tactics/TA0040"
+id = "TA0040"
```

Comment:
I think we can remove the technique here and change the tactic to Exfiltration: https://attack.mitre.org/tactics/TA0010/